### PR TITLE
LT-2977: adjust start date

### PR DESCRIPTION
**Pull request by gponomarev-lykke**

### CODE REVIEW

File: src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs [hunk 1]
```diff
@@ -29,10 +29,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsRequired()
                         .HasColumnType("nvarchar(max)");
 
-                    b.Property<string>("UnderlyingCategoryId")
```
Comment (andrey-anokhin) [inline src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs:32]: This change looks strange and not related to your task

File: src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -164,13 +160,13 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<TimeSpan>("Close")
                         .HasColumnType("time");
 
-                    b.Property<decimal?>("Dividends871M")
+                    b.Property<decimal>("Dividends871M")
```
Comment (andrey-anokhin) [inline src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs:163]: Same for these changes... looks strange and not related to your task

File: src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs (hunk 1 above)
Comment (gponomarev-lykke) [reply src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs]: Fxied

File: src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs (hunk 2 above)
Comment (gponomarev-lykke) [reply src/MarginTrading.AssetService.SqlRepositories/Migrations/AssetDbContextModelSnapshot.cs]: Fxied

**Review by andrey-anokhin — APPROVED** [review]

**Pull request merged**